developing.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+= Developing CodeReady Containers
+
+[[developing-overview]]
+== Overview
+
+The following sections describe how to build the project.
+
+[[develop-prerequisites]]
+== Prerequisites
+
+- Git
+- Make
+- A recent Go distribution (>=1.11)
+
+[NOTE]
+====
+You should be able to develop the project on Linux, Windows, or macOS.
+====
+
+[[set-up-dev-env]]
+== Setting up the development environment
+
+[[cloning-repository]]
+=== Cloning the repository
+
+Get the sources from GitHub:
+
+----
+$ git clone https://github.com/code-ready/crc.git
+----
+
+Note: Do not keep the source code in your $GOPATH as link:https://github.com/golang/go/wiki/Modules[Go modules] will cause the `make` to fail.
```

Comment:
I don't think "Note:" is necessary here unless we want to make this an admonition.

We should mark up `$GOPATH` in monospace.

Minimal rewrite:

```adoc
Do not keep the source code in your `$GOPATH`, as link:https://github.com/golang/go/wiki/Modules[Go modules] will cause `make` to fail.
```